src/lib/client.cc

```diff
@@ -129,8 +129,7 @@ void
 Client::CreateBucket(const QString& name)
 {
 	ds3_request* request = ds3_init_put_bucket(name.toUtf8().constData());
-	LOG_INFO("Create Bucket " + name + " (PUT " + m_endpoint + "/" + \
-		 name + ")");
+	LOG_INFO("PUT          BUCKET    "+m_endpoint+"/"+name);
 	ds3_error* ds3Error = ds3_put_bucket(m_client, request);
```

Comment:
Please follow the prevailing coding style in regards to whitespace.  In this case, there should be spaces around the "+" operators.  Please adjust all other log lines to follow.